From a852bfc1f6459e227cc6fed2fa945a5f4ed9a3dc Mon Sep 17 00:00:00 2001
From: Kalyan Kondapally <kalyan.kondapally@intel.com>
Date: Sat, 10 Sep 2016 22:54:26 -0700
Subject: [PATCH 1/2] Use LOCAL_MODULE_RELATIVE_PATH instead of
 LOCAL_MODULE_PATH.

LOCAL_MODULE_PATH for shared libraries is unsupported in
multiarch builds and LOCAL_MODULE_RELATIVE_PATH should be used
instead.
---
 hdmi/Android.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hdmi/Android.mk b/hdmi/Android.mk
index cef79f2..f9f0b3b 100644
--- a/hdmi/Android.mk
+++ b/hdmi/Android.mk
@@ -21,7 +21,7 @@ ifeq ($(BOARD_USES_TINY_ALSA_AUDIO), true)
 include $(CLEAR_VARS)


-LOCAL_MODULE_PATH := $(TARGET_OUT_SHARED_LIBRARIES)/hw
+LOCAL_MODULE_RELATIVE_PATH := hw
 LOCAL_CFLAGS := -D_POSIX_C_SOURCE=200809

 LOCAL_SRC_FILES := \
--
2.7.4
